### PR TITLE
Adjust to the new lockfile unique lock filename format

### PR DESCRIPTION
**Pull request by eliran-stratoscale**
@rom-stratoscale @michaelk-stratoscale

### CODE REVIEW

File: py/strato/racktest/infra/seed/filecache.py [hunk 1]
```diff
@@ -81,8 +81,11 @@ def _isSameUniqueLock(uniqueFile):
             fullPath = os.path.join(self._cacheDir, uniqueFile)
             return lockFileStat.st_ino == os.stat(fullPath).st_ino
 
-        def _getPidFromUniqueFile(uniqueLockFile):
-            return uniqueLockFile.split('.')[1]
+        def _getPidFromUniqueFile(uniqueLockFile, prefix):
+            PIDAndPrefix = uniqueLockFile.split('.')[-1]
+            assert PIDAndPrefix.endswith(prefix)
```
Comment (rom-stratoscale) [inline py/strato/racktest/infra/seed/filecache.py]: why assert? when will it happen?

Comment (rom-stratoscale) [inline py/strato/racktest/infra/seed/filecache.py]: endswith(prefix) ? shouldnt it be suffix? :)

Comment (rom-stratoscale) [inline py/strato/racktest/infra/seed/filecache.py]: @eliran-stratoscale

Comment (eliran-stratoscale) [inline py/strato/racktest/infra/seed/filecache.py]: Always. Maybe i'll change it so the method will receive the lock itself so it's more clear.

Comment (eliran-stratoscale) [inline py/strato/racktest/infra/seed/filecache.py]: Yeah, my bad.

**Comment by rom-stratoscale** [comment]
@eliran-stratoscale how do u suggest we test it? Run a monkey cycle?

**Comment by eliran-stratoscale** [comment]
Yes. plus, i've extended its UT.

File: py/strato/racktest/infra/seed/filecache.py [hunk 2]
```diff
@@ -51,13 +51,13 @@ def lock(self, key):
                 lock.acquire(timeout=60)
                 break
             except lockfile.LockTimeout:
-                logging.warning('Lock timed out for key %(key)s try to heal cache', dict(key=key))
+                logging.warning('Lock timed out for key %(key)s.', dict(key=key))
+                sanitizedKey = key.hash
```
Comment (michaelk-stratoscale) [inline py/strato/racktest/infra/seed/filecache.py]: already declared and assigned

Comment (eliran-stratoscale) [inline py/strato/racktest/infra/seed/filecache.py]: Fixed.

**Comment by michaelk-stratoscale** [comment]
Merged. Updated dependency in monkey.

**Pull request closed without merging**